src/Plugins/ComplexCore/src/ComplexCore/Filters/RawBinaryReaderFilter.hpp

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "ComplexCore/ComplexCore_export.hpp"
+
+#include "complex/Filter/FilterTraits.hpp"
+#include "complex/Filter/IFilter.hpp"
+
+namespace complex
+{
+/**
+ * @class RawBinaryReaderFilter
+ * @brief This filter will ....
```

Comment:
We should probably copy over a brief description from the markdown documentation.